### PR TITLE
Use aebytecode repository and remove apps/aebytecode

### PR DESCRIPTION
**Pull request by ThomasArts**

### CODE REVIEW

**Comment by lucafavatella** [comment]
Are these two lines necessary now?
```
.dockerignore:32:apps/aebytecode/src/aeb_asm_scan.erl
.gitignore:41:apps/aebytecode/src/aeb_asm_scan.erl
```

File: apps/aebytecode/src/aebytecode.app.src [hunk 1]
```diff
@@ -1,15 +0,0 @@
-{application, aebytecode,
- [{description, "Bytecode definitions for AEthernity VM shared with compiler."},
-  {vsn, "0.1.0"},
```
Comment (lucafavatella) [inline apps/aebytecode/src/aebytecode.app.src]: I see now 1.0.0 - ok.

File: apps/aebytecode/src/aebytecode.app.src [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-  {registered, []},
-  {applications,
-   [kernel,
-    stdlib,
-    lager
```
Comment (lucafavatella) [inline apps/aebytecode/src/aebytecode.app.src]: I see lager removed among applications: intended?

File: apps/aebytecode/src/aebytecode.app.src [hunk 3]
```diff
@@ -1,15 +0,0 @@
-{application, aebytecode,
- [{description, "Bytecode definitions for AEthernity VM shared with compiler."},
```
Comment (lucafavatella) [inline apps/aebytecode/src/aebytecode.app.src]: aeternity - though cosmetic.

**Comment by ThomasArts** [comment]
> Are these two lines necessary now?
> 
> ```
> .dockerignore:32:apps/aebytecode/src/aeb_asm_scan.erl
> .gitignore:41:apps/aebytecode/src/aeb_asm_scan.erl
> ```

Yes, this is a generated file from aeb_asm_scan.xrl

File: apps/aebytecode/src/aebytecode.app.src (hunk 3 above)
Comment (ThomasArts) [reply apps/aebytecode/src/aebytecode.app.src]: I just copied it ;-). Will make a new PR for that... not so urgent.

File: apps/aebytecode/src/aebytecode.app.src (hunk 2 above)
Comment (ThomasArts) [reply apps/aebytecode/src/aebytecode.app.src]: Yes, this one is not depending on lager... it's just a very small library

File: apps/aebytecode/src/aebytecode.app.src (hunk 1 above)
Comment (ThomasArts) [reply apps/aebytecode/src/aebytecode.app.src]: The first running version, already in mainnet

**Comment by lucafavatella** [comment]
> Yes, this is a generated file from aeb_asm_scan.xrl

@ThomasArts https://github.com/aeternity/epoch/pull/1977

**Pull request merged**